src/components/project/givbacks/steps/FormField.tsx

```
@@ -0,0 +1,34 @@
+  required?: boolean
+  error?: string
+  hint?: string
+  children: ReactNode
+}) {
+  return (
+    <div>
+      <label className="mb-1 block text-sm font-semibold text-giv-neutral-900">
+        {label} {required && <span className="text-red-500">*</span>}
+      </label>
+      {children}
+      {hint && !error && <p className="mt-1 text-xs text-giv-neutral-500">{hint}</p>}
```

ESLint

[error] Replace `<p·className="mt-1·text-xs·text-giv-neutral-500">{hint}</p>` with `(⏎········<p·className="mt-1·text-xs·text-giv-neutral-500">{hint}</p>⏎······)`

(prettier/prettier)